_posts/2018-06-28-nabla-setup.md

```diff
@@ -1,6 +1,10 @@
 ---
 layout: post
 title: Running a Nabla Container
+date: 2018-06-28
+image: nabla-build-runtime.png
```

Comment:
Is this the thumbnail used for when putting a link? How big in general is this going to be?

Could we default to the nabla logo: https://github.com/nabla-containers/nabla-containers.github.io/blob/master/public/img/nabla.png? So that at least we'll have something for the default behavior?